libxext will fail pkgconf test pipeline

If the pkgconf test pipeline is added to libxext it will fail in the following way:

```
2024/11/12 23:12:42 INFO running step "test/pkgconf"
2024/11/12 23:12:42 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:12:42 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:12:42 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:12:42 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:12:42 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:42 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:42 WARN + basename /home/build/melange-out/libxext-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:42 WARN + dev_pkg=libxext-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:42 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:42 WARN + apk info -L libxext-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:42 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:42 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:42 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:42 WARN + grep -q ^Name: usr/lib/pkgconfig/xext.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:42 WARN + basename usr/lib/pkgconfig/xext.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:42 WARN + lib_name=xext uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:42 WARN + echo usr/lib/pkgconfig/xext.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:42 WARN + grep -q '^usr/lib/pkgconfig/xext.pc$\|^usr/share/pkgconfig/xext.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:42 WARN + pkgconf --exists xext uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:42 WARN + grep -q ^Version: usr/lib/pkgconfig/xext.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:42 WARN + pkgconf --modversion xext uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:42 INFO 1.3.6 uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:42 WARN + grep -q ^Libs: usr/lib/pkgconfig/xext.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:42 WARN + pkgconf --libs xext uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:42 INFO -lXext uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:42 WARN + grep -q ^Cflags: usr/lib/pkgconfig/xext.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:42 WARN + pkgconf --cflags xext uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:42 WARN Package x11 was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:42 WARN Perhaps you should add the directory containing `x11.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:42 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:42 WARN Package 'x11', required by 'xext', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:42 INFO pod 3ba3f0fa3c2952f597c5818c1874ef72795e7f50e9626e5bfd36f87b6df4ba2a terminated
2024/11/12 23:12:42 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:12:42 INFO   workspace dir: /home/user/tmp/melange-workspace-2577782323
2024/11/12 23:12:42 INFO   guest dir: /home/user/tmp/melange-guest-3537173720
2024/11/12 23:12:42 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/libxext] Error 1
```

--- a/libxext.yaml
+++ b/libxext.yaml
@@ -1,7 +1,7 @@
 package:
   name: libxext
   version: 1.3.6
-  epoch: 3
+  epoch: 4
   description: X11 miscellaneous extensions library
   copyright:
     - license: MIT
@@ -52,7 +52,11 @@ subpackages:
       runtime:
         - libxext
         - libxau-dev
+        - libx11-dev
     description: libxext dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
 update:
   enabled: true

